datalake/src/test/java/com/sequenceiq/datalake/service/sdx/StorageValidationServiceTest.java

```diff
@@ -14,13 +17,20 @@
 import com.sequenceiq.common.api.cloudstorage.old.GcsCloudStorageV1Parameters;
 import com.sequenceiq.common.api.cloudstorage.old.S3CloudStorageV1Parameters;
 import com.sequenceiq.common.model.FileSystemType;
+import com.sequenceiq.datalake.entity.SdxCluster;
 import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
 import com.sequenceiq.sdx.api.model.SdxCloudStorageRequest;
 import com.sequenceiq.sdx.api.model.SdxClusterRequest;
 
 @ExtendWith(MockitoExtension.class)
 class StorageValidationServiceTest {
 
+    @Mock
+    EnvironmentService environmentService;
+
+    @Mock
+    SdxCluster sdxCluster;
```

Comment:
Nit: Visibility could be reduced to `private`.

Reply:
done